src/platform/FreeBSD.hh

```diff
@@ -32,6 +32,7 @@
 
 #include "platform/Capsicum.hh"
 #include "File.hh"
+#include<libpreopen.h>
```

Comment:
For the purposes of declaring a pointer field, you can just use a forward declaration (`struct po_map;`) rather than including `libpreopen.h` in any file that includes `platform/FreeBSD.hh`.